src/commands/run.ts

```diff
@@ -176,45 +177,26 @@ export class RunCommand extends ApifyCommand<typeof RunCommand> {
 		let CRAWLEE_PURGE_ON_START = '0';
 
 		// Purge stores
-		// TODO: this needs to be cleaned up heavily - ideally logic should be in the project analyzers
 		if (this.flags.purge) {
-			switch (projectType) {
-				case PROJECT_TYPES.PRE_CRAWLEE_APIFY_SDK: {
-					await Promise.all([purgeDefaultQueue(), purgeDefaultKeyValueStore(), purgeDefaultDataset()]);
-					info({ message: 'All default local stores were purged.' });
-					break;
-				}
-				case PROJECT_TYPES.CRAWLEE:
-				default: {
-					CRAWLEE_PURGE_ON_START = '1';
-				}
+			// Crawlee does auto purging
+			if (projectTypes.includes(PROJECT_TYPES.CRAWLEE)) {
```

Comment:
looks like this wont detect SDK v3 without crawlee explicitly installed, which should have this behaviour too

Reply:
Well, SDK v3 usually also has crawlee defined as a dependency in *some* way, so it will match, no? But then again, if someone installs just SDK v3, and doesn't use crawlee at all, we wouldn't purge, no?

Reply:
so this will catch transitive dependencies too? you can use the SDK without explicitly installing crawlee, that's my concern

Reply:
> and doesn't use crawlee at all, we wouldn't purge, no?

wdym? SDK v3 should behave just like crawlee project, it also autopurges

Reply:
*if* you use crawlee crawlers, otherwise they don't, no? iirc our purging logic only runs when starting a crawler?

Reply:
no, you dont have to use crawlers at all, same with JS variant

Reply:
auto purge is implemented on storage level 

https://github.com/apify/crawlee/blob/f07cac167a8157bba6db1f7b7138085b74e27f81/packages/core/src/storages/dataset.ts#L634

Reply:
Oh right, we did it on `open`. eek, ok, will need to adjust the analyzer

Reply:
Whats the version for python SDK, or does that diverge from crawlee for storages?

Reply:
SDK v2 should work the same, thats where we used python crawlee internally (which is responsible for this)